### DISCUSSION TITLE
How to change the default keys

### DISCUSSION THREAD

**Discussion by Zuash** [flameshot-org/flameshot#2915] [Q&A] [answered]
Some keys cant change in the menu bar. I want to change the screenshot button (remove from prn scr) to another combination. os Windows

**Comment by mmahmoudian** [comment, marked as answer]
Please search in the issues before creating a new one. It save you a lot of time (you get the answer immediately, and also it helps avoiding duplicates).

Duplicate of  #1341 , #2887 , and many others.

Read this for a workaround:

https://github.com/flameshot-org/flameshot/issues/1861#issuecomment-910552032

As a personal note, Windows is not as customizable as Linux, so if you choose to use Windows, it comes with certain limitations. This does not mean that Flameshot cannot improved further, but it means that the developer should do it for you rather than you being in full control of your computer.

Anyways, I hope the solution above helps.
